src/SkynetServer.Shared/Services/ConfirmationMailService.cs

```diff
@@ -10,15 +10,22 @@
 
 namespace SkynetServer.Services
 {
-    public class MailingService
+    public class ConfirmationMailService
     {
         private readonly IOptions<MailOptions> mailOptions;
+        private readonly IOptions<WebOptions> webOptions;
 
-        public MailingService(IOptions<MailOptions> mailOptions)
+        public ConfirmationMailService(IOptions<MailOptions> mailOptions, IOptions<WebOptions> webOptions)
         {
             this.mailOptions = mailOptions;
+            this.webOptions = webOptions;
         }
 
+        public Uri GetConfirmationUrl(string token)
+        {
+            return new Uri(webOptions.Value.PublicBaseUrl, "confirm/" + token);
```

Comment:
If we are going to make it configurable, why not make it fully configurable, so that one can even do something like https://confirm.skynet.app/{token}

Reply:
The `SkynetServer.Web` project is designed to handle every account specific user interaction from web browsers. As far as now only confirmation mails are implemented but other features like deleting your account if you forgot your password will also be handled by this application.

In my opinion we can still add separate options for every page if we really need it.